hdd_tools/config.json

```diff
@@ -10,17 +10,25 @@
   "init": false,
   "map": ["share:rw"],
   "privileged": ["SYS_ADMIN", "SYS_RAWIO"],
+  "devices": ["/dev/sda", "/dev/sdb", "/dev/sdc", "/dev/sdd", "/dev/sde", "/dev/nvme0", "/dev/nvme1"],
   "full_access": true,
-  "homeassistant": "2021.2.0",
+  "homeassistant": "2024.11.3",
   "homeassistant_api": true,
   "options": {
       "sensor_state_type": "temperature",
+      "additional_sensor_state_types": [],
       "sensor_name": "sensor.hdd_smart",
+      "additional_sensor_names": [],
       "friendly_name": "Hdd SMART",
+      "additional_friendly_names": [],
       "hdd_path": "/dev/sda",
+      "additional_hdd_paths": [],
       "device_type": "auto",
+      "additional_device_types": [],
       "attributes_format": "object",
+      "additional_attributes_formats": [],
       "attributes_property": "",
+      "additional_attributes_properties": [],
```

Comment:
If we can't migrate from old to a new version automatically, at least I think we can "aggrupate" the new disks. I've seen this post: https://community.home-assistant.io/t/optional-array-in-add-on-configuration-option/380109/5

If I understand correctly, we can add a simple `additional_drives` array, and inside it, one element for each drive with the habitual `sensor_state_types`, `sensor_names`, etc. for each one. In this way it seems less intrusive and more similar to what we have (at least in names). 

What do you think?

Reply:
Cool idea. Let me take a look at this during the holidays.